### ISSUE TITLE
Is this still in development?

### ISSUE THREAD

**Issue by ZASMan** [cmj7gh/ClientTracker#17]
Hey @cmj7gh, I was on Github and thought I'd ask, is there any tasks that need to be done on this? I wouldn't mind doing a task or two occasionally.

**Comment by ghost** [comment]
Hey Zane - hope you're doing well! Yes, there's still plenty to be done, but I haven't had much time to work on it lately. 

We're migrating to a new hosting provider soon, and the new server has a more recent version of PHP that I'm worried will have compatibility issues with the code. So I'm hoping to upgrade to CakePHP 3.0 soon, but I've heard that's a big undertaking. 

Once the upgrade and migration are finished, I'll get back to feature development.

**Comment by ZASMan** [comment]
Glad to hear it Chris! Feel free to hit up me or @ashivkum when you get back to feature development, we'd still love to help!

**Comment by ashivkum** [comment]
Yeah, sorry about that, I hadn't heard much from y'all, and so I forgot
about this project over time (work caught up on me).

I'd still be happy to be an IC for this though!
